packages/legend-query-builder/src/stores/__tests__/QueryBuilderChangeDetection.test.ts

```diff
@@ -89,5 +89,11 @@ test(integrationTest('Test change detection'), () => {
     expect(queryBuilderState.hashCode).toBe(
       queryBuilderState.changeDetectionState.hashCodeSnapshot,
     );
+
+    // Test Redo/Undo action in Query Builder
+    fireEvent.click(renderResult.getByText('Undo'));
```

Comment:
can we please add a new test in a new file instead of modifying existing one?

Reply:
sure! I will create a new one. I thought we could  avoid some duplicated code If we could use the existing one for query setup